### PR TITLE
PLA-8072: Correct DesignWorkflow docstring

### PR DESCRIPTION
**Pull request by dkao-citrine**
# Citrine Python PR

## Description 
https://citrine.atlassian.net/browse/PLA-8072

The `DesignWorkflow` class's docstring erroneously listed `project_id` as an initialization argument.

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [x] I have bumped the version in setup.py

### CODE REVIEW

**Review by andyczerwonka — DISMISSED** [review]

File: src/citrine/informatics/workflows/design_workflow.py [hunk 1]
```diff
@@ -24,8 +24,6 @@ class DesignWorkflow(Resource['DesignWorkflow'], Workflow, AIResourceMetadata):
         if none is provided, one matching your design space will be automatically generated
     predictor_id: UUID
         the UUID corresponding to the predictor to use
-    project_id: UUID
```
Comment (talltea) [inline src/citrine/informatics/workflows/design_workflow.py:27]: Nice! If you want to go above and beyond the ticket, the docstring is also missing the `description` param

**Review by talltea — DISMISSED** [review]

**Review by talltea — APPROVED** [review]
thank you!

File: src/citrine/informatics/workflows/design_workflow.py (hunk 1 above)
Comment (dkao-citrine) [reply src/citrine/informatics/workflows/design_workflow.py:27]: Done! I added it below. What do you think?

**Comment by dkao-citrine** [comment]
```
The command "pytest --cov=src/ --cov-report term-missing --cov-report term:skip-covered --cov-config=tox.ini --cov-fail-under=100 -s -r ." exited with 0.
5.21s$ flake8 src
src/citrine/informatics/workflows/design_workflow.py:14:1: D413 Missing blank line after last section
The command "flake8 src" exited with 1.
0.13s$ derp src src/citrine/__version__.py
The command "derp src src/citrine/__version__.py" exited with 0.
```

I added back the newline at the end of the docstring section. The CI tests should pass this time.

**Review by talltea — APPROVED** [review]

**Pull request merged**